ci/release.js

```diff
@@ -45,9 +45,8 @@ pomParser.parse(opts, function(err, response) {
         // Remove release tag
         ci.sh('git push --delete origin ' + gitTag);
         
-        // Remove NPM token and 'gpg' config
+        // Remove NPM token
         ci.sh('rm -f ~/.npmrc');
-        ci.sh('rm -rf /home/circleci/.gnupg');
```

Comment:
I think this folder should still be deleted afterwards, since it contains the GPG profile you import earlier.

Reply:
Good catch, I was a bit too fast. I reverted that change.